RunningOrder/Sprint/Extensions/Sprint+CloudKit.swift

```diff
@@ -17,6 +17,7 @@ extension Sprint: CKRecordable {
         self.number = try record.property("number")
         self.colorIdentifier = try record.property("colorIdentifier")
         self.zoneId = record.recordID.zoneID
+        self.closed = (try? record.property("closed")) ?? false
```

Comment:
could you add a comment to tell that the try? ?? false will be removed in a futur version when all the sprints will be migrated ? :)